.env_file.test

```diff
@@ -7,3 +7,9 @@ S3_ENDPOINT=http://minio:9000
 AWS_DEFAULT_REGION=us-east-1
 AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
 AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+OTEL_SERVICE_NAME=my-service-name
+OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
+OTEL_EXPORTER_OTLP_ENDPOINT="https://api.honeycomb.io"
+OTEL_EXPORTER_OTLP_HEADERS="x-honeycomb-team=XXX"
+HONEYCOMB_API_KEY=XXX
```

Comment:
Does this 'XXX' work with local tests?  I thought it may need to be blank so that it will trigger the logic to not try to contact the external endpoint?

Reply:
ok, will alter